### PR TITLE
Adds some new EUM-centric things

### PR DESCRIPTION
**Pull request by Yonnimer**
Adds origins, accents, religions, clothing, and a consular role for Dionae revolving around EUM

Also adds some options for Skrell from EUM

Ready for review, but **do not merge** yet as this is meant to come with a lore update

### CODE REVIEW

**Comment by Yonnimer** [comment]
!wip

**Comment by Yonnimer** [comment]
!wip

**Comment by Yonnimer** [comment]
!review

**Comment by Yonnimer** [comment]
Went through and tested everything, looks to be working fine

File: code/modules/background/origins/origins/diona/eum.dm [hunk 1]
```diff
@@ -9,6 +9,57 @@
...
+	possible_accents = list(ACCENT_ROOTSONG, ACCENT_VOIDSONG, ACCENT_ANCIENTSONG)
+	possible_citizenships = list(CITIZENSHIP_NRALAKK, CITIZENSHIP_EUM, CITIZENSHIP_BIESEL, CITIZENSHIP_COALITION)
+	possible_religions = list(RELIGION_ETERNAL, RELIGION_GLEANERS, RELIGION_OTHER, RELIGION_NONE)
+	
+/singleton/origin_item/origin/nraldaaq
+	name = "Nral’Daaq"
+	desc = "Playing host to the vast majority of the planet’s non-dionae population, Nral’Daaq is a city-state under the CT-EUM that is known for being the planet’s hub of off-world trading, as well as for its quite diverse majority-skrell population, with both Pro-Federation and Anti-Federation Skrell calling the city home. Due to the city’s border with the sea, Nral’Daaq is also a quite popular trading spot for the Ekane, with the vast majority of their goods ending up in the city to be later resold off-world. "
```
Comment (Alberyk) [inline code/modules/background/origins/origins/diona/eum.dm:32]:
```suggestion
	desc = "Playing host to the vast majority of the planet's non-dionae population, Nral'Daaq is a city-state under the CT-EUM that is known for being the planet's hub of off-world trading, as well as for its quite diverse majority-skrell population, with both Pro-Federation and Anti-Federation Skrell calling the city home. Due to the city's border with the sea, Nral'Daaq is also a quite popular trading spot for the Ekane, with the vast majority of their goods ending up in the city to be later resold off-world. "
```

File: code/modules/background/origins/origins/diona/eum.dm [hunk 2]
```diff
@@ -9,6 +9,57 @@
...
-	possible_accents = list(ACCENT_ROOTSONG, ACCENT_VOIDSONG)
+	possible_accents = list(ACCENT_ROOTSONG, ACCENT_VOIDSONG, ACCENT_ANCIENTSONG)
+	possible_citizenships = list(CITIZENSHIP_NRALAKK, CITIZENSHIP_EUM, CITIZENSHIP_BIESEL, CITIZENSHIP_COALITION)
+	possible_religions = list(RELIGION_ETERNAL, RELIGION_MADA, RELIGION_GLEANERS, RELIGION_OTHER, RELIGION_NONE)
+	
+/singleton/origin_item/origin/united
+	name = "United"
+	desc = "Widely considered the capital of the CT-EUM, the city-state of United is the newest independent nation, formed alongside the formation of the CT-EUM, to act as a neutral meeting place for the Council of Delegates, and to represent the central governing body, as well as acting as a mediator between the two other city-states should a conflict arise. United is the second most-visited city on EUM after Nral’Daaq, and has influence from both off-world and the other city-states in its structure, and operation."
```
Comment (Alberyk) [inline code/modules/background/origins/origins/diona/eum.dm:18]:
```suggestion
	desc = "Widely considered the capital of the CT-EUM, the city-state of United is the newest independent nation, formed alongside the formation of the CT-EUM, to act as a neutral meeting place for the Council of Delegates, and to represent the central governing body, as well as acting as a mediator between the two other city-states should a conflict arise. United is the second most-visited city on EUM after Nral'Daaq, and has influence from both off-world and the other city-states in its structure, and operation."
```

**Review by Alberyk — CHANGES_REQUESTED** [review]

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**